homeassistant/components/plex/__init__.py

```diff
@@ -129,3 +131,20 @@ def _setup_plex(hass, config):
         )
 
     return True
+
+
+async def async_unload_entry(hass, entry):
+    """Unload a config entry."""
+    server_id = entry.data[CONF_SERVER_IDENTIFIER]
+
+    cancel = hass.data[PLEX_DOMAIN][REFRESH_LISTENERS].pop(server_id)
+    await hass.async_add_executor_job(cancel)
+
+    for platform in PLATFORMS:
+        hass.async_create_task(
+            hass.config_entries.async_forward_entry_unload(entry, platform)
```

Comment:
These coroutines should be awaited. We can asyncio.gather to await all concurrently. We don't need to create tasks explicitly.